actions/src/dispatcher.js

```diff
@@ -35,9 +35,11 @@ module.exports = {
         for (var i = 0; i < checkList.length; i++) {
           switch (checkList[i]) {
             case constants.issuesLabelCheck:
+              core.info('issue label check triggered');
```

Comment:
Do we need this?

Reply:
This is to make it clear what is being executed in the GitHub Action. Since we have one workflow for both the checks (Issue label check and CLA Check). Will also make debugging easier in the future.